contrib/net/http/option.go

```diff
@@ -95,6 +99,28 @@ func WithSpanOptions(opts ...ddtrace.StartSpanOption) Option {
 	}
 }
 
+// WithDynamicResourceNamer automatically populates the name of a resource from the Method and URL.Path of the
+// inbound request.
+func WithDynamicResourceNamer() Option {
+	return WithResourceNamer(func(req *http.Request) string {
+		return req.Method + " " + req.URL.Path
+	})
+}
+
+// WithStaticResourceNamer populates the name of a resource with a static string.
+func WithStaticResourceNamer(name string) Option {
+	return WithResourceNamer(func(req *http.Request) string {
+		return name
+	})
+}
```

Comment:
These are not necessary. The user can do this in his own app. We aim to provide minimal tooling to allow the user to do as he pleases, giving him the necessary building blocks.

Reply:
The static variant is used in maintaining the old path and the dynamic variant is used in the new path.  

Let's revisit this pending resolution of whether we care about maintaining the old contract/behavior, because if we do, then we've got to have these implemented anyway and we might as well expose them right?  

Thanks for your prompt attention to the PR!

Reply:
I'd prefer we only have `WithResourceNamer`, just like all the other integrations. Nothing extra.